### PR TITLE
Ambush ruin

### PR DESCRIPTION
**Pull request by 1glitchycent**
Did I do this right?

## About The Pull Request
Adds a new ruin in jungle. Has an M14 and calcium injector, defended by 3 mobs

## Why It's Good For The Game
More jungle ruins

## Changelog
:cl:
add: Added new jungle  ruin
/:cl:

### CODE REVIEW

**Comment by 1glitchycent** [comment]
image

File: whitesands/code/datums/ruins/jungle.dm [hunk 1]
```diff
@@ -14,3 +14,9 @@
 	id = "jungle-monkies"
 	description = "Ook. Ooh Ooh Aah. AAH. OOK OOK OOK. OOK OOK AAH AAAH AAAAAAAAAAAAAAAH!"
 	suffix = "jungle_surface_monkies.dmm"
+
+/datum/map_template/ruin/jungle/solgov_crash
```
Comment (MarkSuckerberg) [inline whitesands/code/datums/ruins/jungle.dm]:
```suggestion
/datum/map_template/ruin/jungle/ambush
```

**Comment by MarkSuckerberg** [comment]
Also, you'll need to mapmerge the map. Ask on discord if you don't know how

**Comment by 1glitchycent** [comment]
Taking break from SS13, will post map file in discord if anyone wants to PR it. If not, I'll come back to it in June.

**Pull request closed without merging**